docs/modules/ROOT/pages/scripts.adoc

```diff
@@ -0,0 +1,44 @@
+= Useful Script Examples
+:sectnums:
```

Comment:
```suggestion
:sectnums:

```
the paragraph below is not showing in the preview, could this empty line be a fix?

Reply:
Indeed.